--- cgdb-0.6.4/doc/cgdb.texinfo~	2007-04-29 12:03:31.578038948 +0200
+++ cgdb-0.6.4/doc/cgdb.texinfo	2007-04-29 12:04:18.527049935 +0200
@@ -29,9 +29,9 @@
 @end quotation
 @end copying
 
-@dircategory Software development
+@dircategory Programming tools:
 @direntry
-* CGDB: (CGDB).                   GNU Ncurses front end to GDB.
+* CGDB: (cgdb).				GNU Ncurses front end to GDB
 @end direntry
 
 @titlepage
