--- cgdb-0.6.1/cgdb/src/cgdb.c~	2006-03-08 00:08:42.000000000 +0100
+++ cgdb-0.6.1/cgdb/src/cgdb.c	2006-03-08 00:10:29.000000000 +0100
@@ -683,10 +683,11 @@
 init_home_dir (void)
 {
   /* Get the home directory */
-  char *home_dir = getenv ("HOME");
+  char *home_dir = getenv ("HOME_ETC");
   const char *cgdb_dir = ".cgdb";
 
   /* Create the config directory */
+  if (!home_dir) home_dir = getenv("HOME");
   if (!fs_util_create_dir_in_base (home_dir, cgdb_dir))
     {
       logger_write_pos (logger, __FILE__, __LINE__,
