--- cgdb-0.7.1/cgdb/cgdb.cpp.orig	2021-09-30 21:53:42.298867456 +0200
+++ cgdb-0.7.1/cgdb/cgdb.cpp	2021-09-30 21:55:01.881769652 +0200
@@ -805,7 +805,8 @@ static void parse_long_options(int *argc
 static int init_home_dir(void)
 {
     /* Get the home directory */
-    char *home_dir = getenv("HOME");
+    char *home_dir = getenv("HOME_ETC");
+    if (!home_dir) home_dir = getenv("HOME");
 
     /* Make sure the toplevel .cgdb dir exists */
     snprintf(cgdb_home_dir, FSUTIL_PATH_MAX, "%s/.cgdb", home_dir);
